src/main/java/org/jitsi/jigasi/transcription/LocalJsonTranscriptHandler.java

```diff
@@ -0,0 +1,353 @@
+     * {@link Transcript.TranscriptEventType#JOIN},
+     * {@link Transcript.TranscriptEventType#LEAVE},
+     * {@link Transcript.TranscriptEventType#RAISE_HAND} or
+     * {@link Transcript.TranscriptEventType#SPEECH}
+     */
+    public final static String JSON_KEY_EVENT_EVENT_TYPE = "event";
+
+    /**
+     * This field stores the time the event took place as a string
+     */
+    public final static String JSON_KEY_EVENT_TIMESTAMP = "timestamp";
+
+    /**
+     * This field stores the  the participant who caused the event as
+     * a Participant object
+     */
+    public final static String JSON_KEY_EVENT_PARTICIPANT = "participant";
+
+    /**
+     * This field stores the alternative JSON objects as a JSON array
+     */
+    public final static String JSON_KEY_EVENT_TRANSCRIPT = "transcript";
+
+    /**
+     * This field stores the language of the transcript as a string
+     */
+    public final static String JSON_KEY_EVENT_LANGUAGE = "language";
+
+    /**
+     * This field stores a unique id for every message as a string.
+     * Can be used to update results if "is_interim" is or was true
+     */
+    public final static String JSON_KEY_EVENT_MESSAGE_ID = "message_id";
+
+    /**
+     * This field stores whether the speech-to-text result is an interim result,
+     * which means it will be updated in the future, as either true or false
+     */
+    public final static String JSON_KEY_EVENT_IS_INTERIM = "is_interim";
+
+    // "alternative" JSON object fields
+
+    /**
+     * This field stores the text of a speech-to-text result as a string
+     */
+    public final static String JSON_KEY_ALTERNATIVE_TEXT = "text";
+
+    /**
+     * This fields stores the confidence of the speech-to-text result as a
+     * number between 0 and 1
+     */
+    public final static String JSON_KEY_ALTERNATIVE_CONFIDENCE = "confidence";
+
+    // "participant" JSON object fields
+
+    /**
+     * This fields stores the name of a participant as a string
+     */
+    public final static String JSON_KEY_PARTICIPANT_NAME = "name";
+
+    /**
+     * This fields stores the (j)id of a participant as a string
+     */
+    public final static String JSON_KEY_PARTICIPANT_ID = "id";
+
+    @Override
+    public TranscriptHandler.Formatter<JSONObject> format()
+    {
+        return new JSONFormatter();
+    }
+
+    @Override
+    public JSONObject formatTranscriptionResult(TranscriptionResult result)
+    {
+        JSONObject object = new JSONObject();
+
+        SpeechEvent event = new SpeechEvent(Instant.now(), result);
+
+        addEventDescriptions(object, event);
+        addAlternatives(object, event);
+
+        return object;
+    }
+
+    @Override
+    public void publish(JSONObject transcript)
+    {
+        String uglyJSON = transcript.toJSONString();
+
+        String prettyJSOn = new GsonBuilder().setPrettyPrinting().create().
+            toJson(new JsonParser().parse(uglyJSON));
+
```

Comment:
I will remove it - but it makes debugging harder as it makes it impossible to read the final json transcript